workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java

```diff
@@ -105,6 +111,27 @@ private Message buildMessage(ActivityExecutorContext<SendMessage> execution) thr
     return builder.build();
   }
 
+  static String extractContent(Object contentObj, Map<String, Object> parameters, MessageService messageService)
+      throws IOException {
+    String content;
+    try {
+      String template = OBJECT_MAPPER.convertValue(contentObj, SendMessage.Content.class)
+          .getTemplate();
+      Map<String, String> params = parameters.entrySet().stream()
+          .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().toString()));
+      File file = new File(template);
+      if (file.getName().endsWith(".ftl")) {
+        content = messageService.templates().newTemplateFromFile(file.getPath()).process(params);
```

Comment:
to access the the file we might want to use the getResource from the context, that will get you a file from the workflows folder

Reply:
You can check how we load attachment file using that, in this executor :)